polylogue/importers/claude_ai.py

```
@@ -50,17 +75,14 @@ def import_claude_export(
-                "Make sure you're using a valid Claude.ai export file (JSON format)."
-            )
-
         output_dir.mkdir(parents=True, exist_ok=True)
         results: List[ImportResult] = []
-        for conv in conversations:
+        for conv in _iter_conversations(convo_path):
+            if not isinstance(conv, dict):
+                raise ValueError(
+                    "Unexpected Claude export format: expected conversations to be objects. "
+                    "Make sure you're using a valid Claude.ai export file (JSON format)."
+                )
```

Ruff (0.15.9)

[warning] Prefer `TypeError` exception for invalid type

(TRY004)